orchestrator/gorc/src/commands/deploy/erc20.rs

```diff
@@ -20,7 +20,7 @@ pub struct Erc20 {
     #[clap(short, long)]
     ethereum_key: String,
 
-    #[clap(short, long, default_value_t = 1.0)]
```

Comment:
Why are we adjusting this?

Reply:
it was a warning being thrown; i don't think our CI clippy step actually cares though so it was kind of arbitrary.